Packages/com.unity.inputsystem/InputSystem/Controls/Processors/AxisDeadzoneProcessor.cs

```diff
@@ -6,15 +6,46 @@
 
 namespace UnityEngine.InputSystem.Processors
 {
+    /// <summary>
+    /// Clamps values to the range given by <see cref="min"/> and <see cref="max"/> and re-normalizes the resulting
+    /// value to [0..1].
+    /// </summary>
+    /// <remarks>
+    /// This processor is registered (see <see cref="InputSystem.RegisterProcessor{T}"/>) under the name "AxisDeadzone".
+    ///
+    /// It acts like a combination of <see cref="ClampProcessor"/> and <see cref="NormalizeProcessor"/>.
+    ///
+    /// <example>
+    /// <code>
+    /// </code>
+    /// // Bind to right trigger on gamepad such that the value is clamped and normalized between
+    /// // 0.3 and 0.7.
+    /// new InputAction(binding: "&lt;Gamepad&gt;/rightTrigger", processors: "axisDeadzone(min=0.3,max=0.7)");
+    /// </example>
+    /// </remarks>
+    /// <seealso cref="StickDeadzoneProcessor"/>
     [Preserve]
-    internal class AxisDeadzoneProcessor : InputProcessor<float>
+    public class AxisDeadzoneProcessor : InputProcessor<float>
```

Comment:
Most of the processors were internal. Precompiled layouts need to be able to just `new` them so I made them public. Also, don't really see a good reason why they should not be.